zipline/algorithm.py

```diff
@@ -1145,11 +1165,14 @@ def cancel_order(self, order_param):
 
     @api_method
     @require_initialized(HistoryInInitialize())
-    def history(self, assets, bar_count, frequency, field, ffill=True):
-        try:
-            assets = list(assets)
-        except TypeError:
-            assets = [assets]
+    def history(self, bar_count, frequency, field, ffill=True):
+        warnings.warn(
+            "The `history` method is deprecated.  Use `data.history` instead.",
+            category=ZiplineDeprecationWarning,
+            stacklevel=4
```

Comment:
Sorry, I should have been clearer.  If an algo in zipline has:

```python
def handle_data(self, data):
    self.history(...)
```
then probably it should be `stacklevel=1`.  Trying to understand which usage this is for and if we care about any others.